dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/trackedentity/TrackedEntityRequestParamsMapper.java

```diff
@@ -91,7 +91,8 @@ public TrackedEntityOperationParams map(
         validateDeprecatedParameter(
             "ouMode", requestParams.getOuMode(), "orgUnitMode", requestParams.getOrgUnitMode());
 
-    orgUnitMode = validateOrgUnitMode(orgUnitUids, orgUnitMode);
+    orgUnitMode =
+        validateOrgUnitMode(orgUnitUids, orgUnitMode, requestParams.getTrackedEntities(), true);
```

Comment:
And maybe this could be rename to `validateOrgUnitModeForTrackedEntities` or something similar